docs/explanations/principles.md

```diff
@@ -0,0 +1,106 @@
+
+
+# Gutenberg Principles
+
+This document has been written so that those contributing to the Gutenberg project can have some understanding of what guides decisions around what is included in the project (or not).
+
+An important thing to keep at top of mind when reviewing these principles is that they are not isolated from each other. There is a cohesiveness and complementary nature between them that results in a synergistic impact and direction.
+
+When applied, sometimes these principles build on each other much like the idea of constructing a house with a foundation, walls and roof. However, other times the application of the principles results in something that is more similar to a tapestry where threads of each principle are embedded together and evident in the finished feature or design of the project.
+
+
+## Aim for delightful, consistent, accessible user interfaces for empowering content creation and expression.
+
+_“Optimize for the user” (Matias - [Gutenberg or the Ship of Theseus](https://matiasventura.com/post/gutenberg-or-the-ship-of-theseus/))_
+
+Each of the words in this principle are deliberate:
+
+
+### Delightful
+
+The user interfaces should “get out of the way” and support the user in creating their content. They should lead to experiences where the user (in the vernacular of Marie Kondo) _finds delight_ with each new thing they discover they can do or accomplish.
+
+
+### Consistent
+
+Any interfaces are consistently applied. Users should be able to _rely on muscle memory and learned patterns_ as they become familiar with how it works.
+
+This also applies to learning _where to find_ different elements that the user might interact with in creating their content.
+
+
+### Accessible
```

Comment:
This one feels a bit odd as it's more of a general principle of WordPress at wide and not specific to Gutenberg. If anything, it should build upon the two other axis of "canvas for expression" and "uniform yet diverse" to embrace the fact users interacting with the software have distinct characteristics and preferences for how to use said software.

Reply:
There's also something important missing here which is how the software can help users build better websites that are semantic and more accessible. That is, _respect for the viewers as well as the creators_.

Reply:
> That is, respect for the viewers as well as the creators.

This is definitely something that isn't capture well in the current iteration that should be included ❤️ 

> This one feels a bit odd as it's more of a general principle of WordPress at wide and not specific to Gutenberg. If anything, it should build upon the two other axis of "canvas for expression" and "uniform yet diverse" to embrace the fact users interacting with the software have distinct characteristics and preferences for how to use said software.

While I agree it's a general principle of WordPress, I also think it's important to call out the distinctive nature of accessibility in the context of Gutenberg. I like how you reflect on describing this as software accommodating users with "distinct characteristics and preferences for how to use said software". An example of this is reflected in the numerous entry points people have to insert a block. Is there some specific concern here around the use of the word "accessible" itself here?

Reply:
See f8a44af.